app/jobs/sync/zendesk/start_job.rb

```diff
@@ -0,0 +1,6 @@
+class Sync::Zendesk::StartJob < ApplicationJob
+  def perform
+    # Make sure there's no overlap for failed jobs a day ago.
+    Sync::Zendesk::TicketsToNotes.perform_async(0, 3.days.ago.to_date.to_s)
```

Comment:
This will result in an infinite loop correct? Because we'll call this and the job will call itself and we'll eventually be compounding jobs

Reply:
nvm, missed the if statement